app/controllers/devise/two_factor_authentication_controller.rb

```diff
@@ -34,8 +34,6 @@ def reauthn_param
 
     def handle_valid_delivery_method(method)
       send_user_otp(method)
-      resent_message = t("notices.send_code.#{method}")
-      flash[:success] = resent_message if session[:code_sent].present?
```

Comment:
We aren't showing it anymore because we are duplicating the information below. @esgoodman suggested removing it and updating the copy. It sounds like we need some confirmation around that.